### ISSUE TITLE
Output on Union Type with Enum

### ISSUE THREAD

**Issue by sabinadams** [jasonkuhrt/molt#143]
I have Command set up as follows:

```ts
const args = Command.parameters({
  "t template": z.string(),
  "i install": z
    .boolean()
    .or(z.enum(["npm", "yarn", "pnpm"]))
    .default(false),
  "n name": z.string(),
  "p path": z.string(),
}).parse();
```

In the outputted docs I was expecting the **Type** field to show an output similar to the one described here which describes the types in the union as well as the individual options in the enum:
https://github.com/jasonkuhrt/molt/issues/107#issuecomment-1366319040

Probably not a bug per-se, but is the output I receive expected? 
#### Screenshot
<img width="1512" alt="CleanShot 2023-03-07 at 13 44 48@2x" src="https://user-images.githubusercontent.com/18456526/223440387-a1c47794-a58e-4d5d-ade1-61bced1e1045.png">

**Comment by jasonkuhrt** [comment]
@sabinadams Yep there's some info about this rendering here: https://github.com/jasonkuhrt/molt/tree/main/packages/@molt/command#union. 

I will add a setting to explicitly control the rendering.
